Gruntfile.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
+const webpack = require("webpack");
+const webpackConfig = require("./webpack.config");
+const merge = require("webpack-merge");
+const webpackConfigRelease = [];
+
+webpackConfig.forEach(function(config) {
+    webpackConfigRelease.push(merge(config, {
+        devtool: false,
+        plugins: [ new webpack.optimize.UglifyJsPlugin() ]
+    }));
+});
+
+module.exports = function(grunt) {
+    const pkg = grunt.file.readJSON("package.json");
+    grunt.initConfig({
+
+        watch: {
+            updateWidgetFiles: {
+                files: [ "./dist/tmp/src/**/*" ],
+                tasks: [ "webpack:develop", "file_append", "compress:dist", "copy:distDeployment", "copy:mpk" ],
+                options: {
+                    debounceDelay: 250,
+                    livereload: true
+                }
+            },
+            sourceFiles: {
+                "files": [ "./src/**/*" ],
+                "tasks": [ "copy:source" ]
+            }
+        },
+
+        compress: {
+            dist: {
+                options: {
+                    archive: "./dist/" + pkg.version + "/" + pkg.widgetName + ".mpk",
+                    mode: "zip"
+                },
+                files: [ {
+                    expand: true,
+                    date: new Date(),
+                    store: false,
+                    cwd: "./dist/tmp/src",
+                    src: [ "**/*" ]
+                } ]
+            }
+        },
+
+        copy: {
+            distDeployment: {
+                files: [ {
+                    dest: "./dist/MxTestProject/deployment/web/widgets",
+                    cwd: "./dist/tmp/src/",
+                    src: [ "**/*" ],
+                    expand: true
+                } ]
+            },
+            mpk: {
+                files: [ {
+                    dest: "./dist/MxTestProject/widgets",
+                    cwd: "./dist/" + pkg.version + "/",
+                    src: [ pkg.widgetName + ".mpk" ],
+                    expand: true
+                } ]
+            },
+            source: {
```

Comment:
what should be removed ? i removed ` expand: true` and copy:mpk can not finish because it cant find the mpk